assets/stylesheets/layout/_main.scss

```diff
@@ -291,10 +291,10 @@ $sidebar-width-min: 228px;
 	transition: all 0.15s ease-in-out;
```

Comment:
There's still [a reference to `.wpcom-sidebar`](https://github.com/Automattic/wp-calypso/blob/ee7daccb1aa7922d1ba6b97c3df1258dd8502b3e/assets/stylesheets/layout/_main.scss#L318-L320) in this file, which I assume should be removed/updated?